packages/offix-datastore/package.json

```diff
@@ -52,6 +53,7 @@
     "tiny-invariant": "1.1.0",
     "urql": "1.9.8",
     "uuid": "8.3.0",
+    "websql": "^1.0.0",
```

Comment:
So library says that this is only for the node.js which will increase size of the package. 

https://www.npmjs.com/package/websql

Alternatives we can play with 
https://github.com/gdbate/lib-websql

https://www.npmjs.com/package/@types/sqlite3 (and sqllite3 as peerDependency)

Reply:
@wtrocki Oh okay cool, will have a look at this, this might be what I need.

I sorted out the types stuff (I forgot to add them to to `tsconfig`)